app/video_routes.py

```diff
@@ -0,0 +1,115 @@
+from app import db
+from flask import Blueprint, jsonify, request
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from dotenv import load_dotenv
+load_dotenv()
+
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+
+@videos_bp.route("", methods=["GET"])
+def get_video():
+    videos = Video.query.all()
+
+    response = []
+    for video in videos:
+        response.append(video.to_dict())
+
+    return jsonify(response), 200
+
+@videos_bp.route("", methods=["POST"])
+def create_video():
+    request_body = request.get_json()
+    try:
+        new_video = Video.from_dict(request_body)
+        db.session.add(new_video)
+        db.session.commit()
+
+        return jsonify(new_video.to_dict()), 201
+
+    except:
+        if "title" not in request_body.keys():
+            response = {
+                "details" : "Request body must include title."
+            }
+        elif "release_date" not in request_body.keys():
+            response = {
+                "details" : "Request body must include release_date."
+            }
+        elif "total_inventory" not in request_body.keys():
+            response = {
+                "details" : "Request body must include total_inventory."
+            }
+        
+        return jsonify(response), 400
+
+@videos_bp.route("/<video_id>", methods=["GET"])
+def get_a_video(video_id):
+    if not video_id.isnumeric():
+        return jsonify(None), 400
+
+    video = Video.query.get(video_id)
+
+    if not video:
+        return jsonify({"message" : f"Video {video_id} was not found"}), 404
+
+    return jsonify(video.to_dict()), 200    
+
+
+    
+@videos_bp.route("/<video_id>", methods=["DELETE"])
+def delete_a_video(video_id):
+    video = Video.query.get(video_id)
+    
+    if not video:
+        return jsonify({"message" : f"Video {video_id} was not found"}), 404
+    
+    else:
+        db.session.delete(video)
+        db.session.commit()
+        return jsonify(
+        {
+        "id": video.video_id, #this worked but video_id didnt??
+        'details': (f'Video {video.video_id} successfully deleted')
+        }
+
+        ), 200
```

Comment:
The `else` clause isn't necessary here, especially if we're following the guard clause pattern. We can think of `if` statements as `if and only if this expression is true then execute this code, otherwise move on to the rest of the function body`.

```suggestion
    if not video:
        return jsonify({"message" : f"Video {video_id} was not found"}), 404
    
 
    db.session.delete(video)
    db.session.commit()
    return jsonify(
    {
    "id": video.video_id, #this worked but video_id didnt??
    'details': (f'Video {video.video_id} successfully deleted')
    }

    ), 200
```